### DISCUSSION TITLE
how to config superset placed inside iframe

### DISCUSSION THREAD

**Discussion by wangmian123** [apache/superset#24590] [Q&A / Help]
I deployed a superset and create a client in keycloak. I have another app（use the JS adapter）in the same realm . Now I want to open a page from superset inside an iframe in my app. When the superset page is opened，it will redirect to the login page. When I re-enter the account and password, display not authorized.

I found the following information：
Compared with a successful superset login, the cookie does not contain a session when redirecting to the http://superset_domain/oidc_callback?state.... after loging in iframe superset page.

SupersetConfiguration：
I configure superset and keycloak based on the following tutorial：https://github.com/apache/superset/discussions/13915

Configurations I've tried：
I've tried to put superset's configuration SESSION_COOKIE_SAMESITE: 'None'

I have two questions：
1、How to share SSO status between my app and superset（in iframe）
2、Cookie in call_back endpoint is missing after successful login. How to configure it？
Do you have any idea about the cause of this problem?

**Comment by wangmian123** [comment]
Okay, I already know. The superset needs to modify the flask's SECTION_COOKIE_SAMESITE: 'None'

**Reply by dhananjay-kaushik** [reply]
I'm stuck with same issue on chrome only...
Followed all the steps you have

Kindly reach out for help

**Reply by lokeshMrudhul** [reply]
@dhananjay-kaushik its working fine in Firefox, getting same issue in firefox
